t/t2800-jobs-instance-info.t

```diff
@@ -22,14 +22,15 @@ waitfile="${SHARNESS_TEST_SRCDIR}/scripts/waitfile.lua"
 #
 test_expect_success 'start a set of Flux instances' '
 	id=$(flux submit flux start /bin/true) &&
-	id2=$(flux submit -n2 -c1 flux start \
+	id2=$(flux submit -N2 -n2 -c1 flux start \
 		"flux run /bin/false ; \
 		 flux submit --cc=1-4 sleep 300 && \
 		 touch ready && \
 		 flux queue idle") &&
 	flux submit sleep 600 &&
 	flux job wait-event $id clean &&
-	$waitfile -t 60 ready
+	$waitfile -t 60 ready &&
+	flux proxy $id2 flux resource list
```

Comment:
why the additional `flux resource list`?  left over debugging?

Reply:
Yeah, that should probably be in a `test_debug`